src/ex4/server.js

```diff
@@ -0,0 +1,57 @@
+// Express boilerplate, hosting the `dist` file, connecting to the routes
+const fs = require("fs");
+const express = require("express");
+
+require("express-async-errors");
+
+const livereload = require("livereload");
+const connectLiveReload = require("connect-livereload");
+
+const logger = require("./server/middleware/logger");
+
+const errorHandler = require("./server/middleware/error_handler");
+const itemRouter = require("./server/routes/itemRouter");
+const port = 8080;
+const server = express();
+
+const liveReloadServer = livereload.createServer();
+
+server.use(connectLiveReload());
+
+//
+server.use([logger, express.json(), express.static("dist")]);
```

Comment:
You should separate business logic when possible.
logger has nothing to do with the `express.json()` or the `static` so it should be attached in a different line
`server.use(logger);`
`server.use([express.json(), express.static("dist")]);`